Toggl.iOS/Cells/Reports/ReportsDonutChartLegendCollectionViewCell.cs

```diff
@@ -54,7 +55,9 @@ public void SetElement(ReportProjectsDonutChartLegendItemElement element, bool l
 
             ClientLabel.Hidden = element.Client == string.Empty;
 
-            var color = new Color(element.Color).ToNativeColor();
+            var color = Color.ParseAndAdjustToLabel(
+                element.Color, ActiveTheme.Is.DarkTheme
+            ).ToNativeColor();
             ProjectLabel.TextColor = color;
             CircleView.BackgroundColor = color;
```

Comment:
We need to adjust the color dot to dark mode as well.

Reply:
It should use the adjusted color and not the label color is what you mean?

Reply:
Oh, it's not clear what I was referring, sorry about that. I meant this:
```suggestion
            CircleView.BackgroundColor = Color.ParseAndAdjustToUserTheme(
                element.Color, ActiveTheme.Is.DarkTheme
            ).ToNativeColor();
```